packages/eslint-config-nimble-react/lib/rules/base.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = {
+  env: {
+    es6: true,
+    browser: true,
+    node: true,
+    jest: true,
+  },
+  extends: ['@nimblehq/eslint-config-nimble'],
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  rules: {
+    'no-use-before-define': 'off',
+    'no-unused-vars': 'off',
```

Comment:
Is there any specific reason wh\y we keep this rule? Or is it just because the `jsx` config requires `import { React } ...` ( React to be in scope even if unused)? 

Because I think having an error on `unused variable` might be valuable in many situations... 🤔

Reply:
It makes sense. Should we also remove these 3 custom rules? I think the current is not the best practice 🤔 
What do you think? @carryall @malparty

Reply:
@hoangmirs @malparty I think we have these in the Varun project because we added Typescript rules
- [`@typescript-eslint/no-use-before-define`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-use-before-define.md)
- [`@typescript-eslint/no-unused-vars`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unused-vars.md)
- [`@typescript-eslint/no-shadow`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-shadow.md)

so as this would be used on general React projects which we're not sure that would be in Typescript or not so we better remove it (or maybe we can have a separate package for React Typescript later)

Reply:
But we are using the typescript in almost projects, and there are the rules for React then I think we can keep it 💭

Reply:
I added a small comment to explain this in https://github.com/nimblehq/eslint-config-nimble/pull/32/commits/e56eff6eef20fe9f3afcef5c42b4a08426377366

Reply:
Ahhh ok. Yes, if it is TypeScript recommendations then I'm good with the comment~ 
That reminds me we still need to update the compass so that our position about TypeScript are reflected :D